tests/test_tztrout.py

```diff
@@ -0,0 +1,433 @@
+        [
+            # United States -- Special cases to make sure ET is not counted as part of state timezone
+            ('US', 'WI', None, ['+14143334444'], 'CT', True),
+            ('US', 'TX', None, ['+12143334444'], 'CT', True),
+            # United States
+            (
+                'US',
+                'NY',
+                'New York',
+                ['+12123334444', '+16463334444'],
+                'ET',
+                True,
+            ),
+            ('US', 'CA', 'Los Angeles', ['+18183334444'], 'PT', True),
+            ('US', 'IL', 'Chicago', ['+16303334444'], 'CT', True),
+            ('US', 'TX', 'Houston', ['+17133334444'], 'CT', True),
+            ('US', 'PA', 'Philadelphia', ['+12153334444'], 'ET', True),
+            ('US', 'AZ', 'Phoenix', ['+16023334444'], 'MT', True),
+            ('US', 'TX', 'San Antonio', ['+12103334444'], 'CT', True),
+            ('US', 'CA', 'San Diego', ['+16193334444'], 'PT', True),
+            ('US', 'TX', 'Dallas', ['+12143334444'], 'CT', True),
+            ('US', 'CA', 'San Jose', ['+14083334444'], 'PT', True),
+            ('US', 'TX', 'Austin', ['+15123334444'], 'CT', True),
+            ('US', 'IN', 'Indianapolis', ['+13173334444'], 'ET', True),
+            ('US', 'FL', 'Jacksonville', ['+19043334444'], 'ET', True),
+            ('US', 'CA', 'San Francisco', ['+14153334444'], 'PT', True),
+            ('US', 'OH', 'Columbus', ['+16143334444'], 'ET', True),
+            ('US', 'NC', 'Charlotte', ['+17043334444'], 'ET', True),
+            ('US', 'TX', 'Fort Worth', ['+16823334444'], 'CT', True),
+            ('US', 'MI', 'Detroit', ['+13133334444'], 'ET', True),
+            ('US', 'TX', 'El Paso', ['+19153334444'], 'MT', False),
+            ('US', 'TN', 'Memphis', ['+19013334444'], 'CT', False),
+            ('US', 'CO', 'Denver', ['+13033334444'], 'MT', True),
+            ('US', 'DC', 'Washington', ['+12023334444'], 'ET', False),
+            # Canada
+            ('CA', 'ON', 'Toronto', ['+14163334444'], 'ET', False),
+            ('CA', 'QC', 'Montreal', ['+15143334444'], 'ET', False),
+            ('CA', 'AB', 'Calgary', ['+14033334444'], 'MT', False),
+            (
+                'CA',
+                'ON',
+                'Ottawa',
+                ['+13433334444', '+16133334444'],
+                'ET',
+                False,
+            ),
+            ('CA', 'AB', 'Edmonton', ['+17803334444'], 'MT', False),
+            ('CA', 'ON', 'Mississauga', ['+12893334444'], 'ET', False),
+            ('CA', 'MB', 'Winnipeg', ['+14313334444'], 'CT', False),
+            ('CA', 'BC', 'Vancouver', ['+16043334444'], 'PT', False),
+            ('CA', 'NS', 'Halifax', ['+19023334444'], 'AT', False),
+            ('CA', 'SK', 'Saskatoon', ['+13063334444'], 'CT', False),
+        ],
+    )
+    def test_major_cities_us_ca(
+        self,
+        country,
+        state,
+        city,
+        phones,
+        tz_name,
+        assert_ids_equal,
```

Comment:
Is `assert_ids_equal` and `is_exact_match` again?

Reply:
I got rid of the need for this variable when splitting out phone and address tests for major cities: 20a80d7